models/webhook.go

```diff
@@ -0,0 +1,80 @@
+package models
+
+import (
+	"errors"
+
+	log "github.com/gophish/gophish/logger"
+)
+
+// Webhook represents the webhook model
+type Webhook struct {
+	Id       int64  `json:"id" gorm:"column:id; primary_key:yes"`
+	Title    string `json:"title"`
```

Comment:
It might be a small thing, but can we call this `Name` instead of `Title`? This would have it match the rest of the models in Gophish. I only noticed it when I was trying to test it out and it was a bit jarring moving from "name" to "title".

Reply:
fixed